packages/fast-jss-manager-angular/build/jestGlobalMocks.ts

```diff
@@ -0,0 +1,16 @@
+const mock = () => {
```

Comment:
file naming seems to be inconsistent with other folders.  in fast-jss-manager-react/build the naming should be consistent. One issue is that is looks like a .js file is being used by react for some reason and shouldn't everything be written in typescript?  If so, then we should file an issue on fast-jss-manager-react to reconcile.